src/app/components/cards/PostSummary.jsx

```diff
@@ -281,13 +280,13 @@ export default connect(
         }
         return {
             post, content, pending_payout, total_payout,
-            username: state.user.getIn(['current', 'username']) || state.offchain.get('account'),
-            blogmode: state.app.getIn(['user_preferences', 'blogmode']),
+            username: state.getIn(['user', 'current', 'username']) || state.getIn(['offchain', 'account']),
+            blogmode: state.getIn(['app', 'user_preferences', 'blogmode']),
         };
     },
 
     (dispatch) => ({
-        dispatchSubmit: data => { dispatch(user.actions.usernamePasswordLogin({...data})) },
-        clearError: () => { dispatch(user.actions.loginError({error: null})) }
+        dispatchSubmit: data => { dispatch({type: 'user/USERNAME_PASSWORD_LOGIN', payload: {...data}}) },
```

Comment:
is this really the right way to namespace events? It means you're going to have to parse them at the reducer, which costs time.

Two alternative proposals:
1. `type` should be an array of namespaces, e.g.:
```
// option 1
dispatch({ type: ['user', 'USERNAME_PASSWORD_LOGIN'] });
// option 2
dispatch({ type: 'user', name: 'USERNAME_PASSWORD_LOGIN' });
```

Thoughts?

Reply:
i'm into option 1... thoughts, others?

Reply:
i don't think there are any huge disadvantages to using an array for action.type but on the flipside, i think it might be nice to follow the ducks pattern -- it's something that many developers are familiar with

Reply:
we can discuss in https://github.com/steemit/condenser/issues/2036